CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * [#7252](https://github.com/rubocop-hq/rubocop/issues/7252): Prevent infinite loops by making `Layout/SpaceInsideStringInterpolation` skip over interpolations that start or end with a line break. ([@buehmann][])
 * [#7262](https://github.com/rubocop-hq/rubocop/issues/7262): `Lint/FormatParameterMismatch` did not recognize named format sequences like `%.2<name>f` where the name appears after some modifiers. ([@buehmann][])
 * [#7253](https://github.com/rubocop-hq/rubocop/issues/7253): Fix an error for `Lint/NumberConversion` when `#to_i` called without a receiver. ([@koic][])
+* Treat `begin`-`end` blocks like `do`-`end` blocks in `Layout/BlockAlignment`. ([@fphilipe][])
```

Comment:
Can you add the following URL to this changelog entry and rebase this PR with latest master branch?

```diff
-* Treat `begin`-`end` blocks like `do`-`end` blocks in `Layout/BlockAlignment`. ([@fphilipe][])
+* [#7286](https://github.com/rubocop-hq/rubocop/pull/7286): Treat `begin`-`end` blocks like `do`-`end` blocks in `Layout/BlockAlignment`. ([@fphilipe][])
```